llvm/include/llvm/Debuginfod/Debuginfod.h

```diff
@@ -46,6 +46,10 @@ bool canUseDebuginfod();
 /// environment variable.
 SmallVector<StringRef> getDefaultDebuginfodUrls();
 
+/// Sets the list of debuginfod server URLs to query. This overrides the
+/// environment variable DEBUGINFOD_URLS.
```

Comment:
Should this override the env vars? Or maybe add to them? Maybe we add a bool parameter to allow the user to choose like `bool append`?

Reply:
Switched the setting to an array of strings setting, which addresses the usage issues.